pkg/plugins/utils/version/filter.go

```diff
@@ -55,6 +61,8 @@ func (f Filter) Init() (Filter, error) {
 		f.Pattern = "*"
 	} else if f.Kind == REGEXVERSIONKIND && len(f.Pattern) == 0 {
 		f.Pattern = ".*"
+	} else if f.Kind == TIMEVERSIONKIND && len(f.Pattern) == 0 {
+		f.Pattern = "2006-01-02"
```

Comment:
Looks good, was wondering is there a reason this date was chosen as default?

Seems random to me, but I might be missing the context.

Reply:
Actually this is a pattern and date itself irrelevant?

Reply:
If that's the case should this default also be applicable for regex time variant?

Reply:
Good catch, I was setting a default pattern for "time" but not "regex/time"
Also, I agree with you that 2006-01-02 is an arbitrarily format which correspond to YYYY-MM-DD
I don't really have strong opinion across the different format like YYYY-MM-DD, YYYY-DD-MM, etc.

Reply:
`YYYY-MM-DD` always feels correct to be region agnostic.